### PR TITLE
Update production playbook

### PR DESCRIPTION
**Pull request by settermjd**
This PR:

- Updates the README, adding extra command-line switches, to make the build instructions more specific.
- Renamed the core Playbook file from site.yml to site.prod.yml.
- Removes the ui.bundle.url setting from the production/core Playbook file.

### CODE REVIEW

File: docs/build-the-docs.md [hunk 1]
```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
...
+        --url <site.url> \
+        site.yml
 ```
+This build command configuration:
 
-This command starts up the Antora Docker container, removes the existing documentation, and then runs Antora's generate command, an regenerates the documentation.
-If all goes well, you will _not_ see any console output.
+- Suppresses all output (`quiet` and `silent`)
+- Remove the output directory before publishing the site (`clean`)
+- Download updates to remote resources (`pull`)
```
Comment (mmattel) [inline docs/build-the-docs.md]: Wouldn´t it be correct to say 
``Download updates to remote resources`` --> ``Download updates from remote resources``

File: docs/build-the-docs.md [hunk 2]
```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
...
+- Specifies a path to the desired UI bundle (`ui-bundle`)
+- Specifies the site URL (`url`)
+
+`ui-bundle` is required, as this isn't specified in the production Playbook file.
+The other options, however, are optional.
+If you've created a custom Playbook file, feel free to use it when running the command, in place of `site.prod.yml`.
```
Comment (mmattel) [inline docs/build-the-docs.md]: I would add at note/text, that ``site.prod.yml`` and ``ui-bundle.zip`` is located in the root of docs.

File: docs/build-the-docs.md [hunk 3]
```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
...
+
+- Suppresses all output (`quiet` and `silent`)
+- Remove the output directory before publishing the site (`clean`)
+- Download updates to remote resources (`pull`)
+- Specifies a path to the desired UI bundle (`ui-bundle`)
+- Specifies the site URL (`url`)
+
+`ui-bundle` is required, as this isn't specified in the production Playbook file.
+The other options, however, are optional.
+If you've created a custom Playbook file, feel free to use it when running the command, in place of `site.prod.yml`.
```
Comment (mmattel) [inline docs/build-the-docs.md]: I would add at note/text, that ``site.prod.yml`` and ``ui-bundle.zip`` is located in the root of docs.

File: docs/build-the-docs.md [hunk 4]
```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
     -v $(pwd):$(pwd) \
     -w $(pwd) \
     antora/antora:1.0.1 \
-    generate site.yml
+    generate \
+        --clean \
+        --pull \
+        --quiet \
+        --silent \
+        --ui-bundle-url <url-or-path-to-ui-bundle>i \
```
Comment (mmattel) [inline docs/build-the-docs.md]: ``--ui-bundle-url <url-or-path-to-ui-bundle>i \`` --> ``--ui-bundle-url <url-or-path-to-ui-bundle.zip> \``

File: docs/build-the-docs.md [hunk 5]
```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
...
 From the command line, in the root of the docs directory, run the command:
 
 ```
-antora site.yml
+antora \
+    --clean \
+    --pull \
+    --quiet \
+    --silent \
+    --ui-bundle-url <url-or-path-to-ui-bundle>i \
```
Comment (mmattel) [inline docs/build-the-docs.md]: ``--ui-bundle-url <url-or-path-to-ui-bundle>i \`` --> ``--ui-bundle-url <url-or-path-to-ui-bundle.zip> \``

File: docs/build-the-docs.md (hunk 1 above)
Comment (settermjd) [reply docs/build-the-docs.md]: Yes. Well spotted.

File: docs/build-the-docs.md (hunk 2 above)
Comment (settermjd) [reply docs/build-the-docs.md]: Sounds fair to me. I'll make that change.

File: docs/build-the-docs.md (hunk 4 above)
Comment (settermjd) [reply docs/build-the-docs.md]: Oh naff, I didn't see that I'd made that mistake. Thanks.

File: docs/build-the-docs.md [hunk 6]
```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
...
 
-This command starts up the Antora Docker container, removes the existing documentation, and then runs Antora's generate command, an regenerates the documentation.
-If all goes well, you will _not_ see any console output.
+- Suppresses all output (`quiet` and `silent`)
+- Removes the output directory before publishing the site (`clean`)
+- Downloads updates from remote resources (`pull`)
+- Specifies a path to the desired UI bundle (`ui-bundle`)
+- Specifies the site URL (`url`)
+
+`ui-bundle` is required, as this isn't specified in the production Playbook file.
```
Comment (mmattel) [inline docs/build-the-docs.md]: ``ui-bundle`` --> ``ui-bundle.zip``

File: docs/build-the-docs.md [hunk 7]
```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
...
-If all goes well, you will _not_ see any console output.
+This build command configuration:
+
+- Suppresses all output (`quiet` and `silent`)
+- Remove the output directory before publishing the site (`clean`)
+- Download updates to remote resources (`pull`)
+- Specifies a path to the desired UI bundle (`ui-bundle`)
+- Specifies the site URL (`url`)
+
+`ui-bundle` is required, as this isn't specified in the production Playbook file.
```
Comment (mmattel) [inline docs/build-the-docs.md]: ``ui-bundle`` --> ``ui-bundle.zip``

File: docs/build-the-docs.md [hunk 8]
```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
...
 ```
+This build command configuration:
 
-This command starts up the Antora Docker container, removes the existing documentation, and then runs Antora's generate command, an regenerates the documentation.
-If all goes well, you will _not_ see any console output.
+- Suppresses all output (`quiet` and `silent`)
+- Removes the output directory before publishing the site (`clean`)
+- Downloads updates from remote resources (`pull`)
+- Specifies a path to the desired UI bundle (`ui-bundle`)
+- Specifies the site URL (`url`)
```
Comment (mmattel) [inline docs/build-the-docs.md]: What is the site URL,

can you given an example below?

File: docs/build-the-docs.md [hunk 9]
```diff
@@ -19,21 +19,53 @@ docker run -ti --rm \
...
 
 ```
-antora site.yml
+antora \
+    --clean \
+    --pull \
+    --quiet \
+    --silent \
+    --ui-bundle-url <url-or-path-to-ui-bundle.zip> \
+    --url <site.url> \
```
Comment (mmattel) [inline docs/build-the-docs.md]: What is the site URL,

can you given an example below?

File: docs/build-the-docs.md (hunk 6 above)
Comment (mmattel) [reply docs/build-the-docs.md]: Why didnt you add the note like you did for site.prod.yml that ui-bundlie can be found in root, is there a specific reason?

File: docs/build-the-docs.md (hunk 7 above)
Comment (mmattel) [reply docs/build-the-docs.md]: Why didnt you add the note like you did for site.prod.yml that ui-bundle.zip can be found in root, is there a specific reason?

**Pull request merged**